src/index.d.ts

```diff
@@ -94,7 +94,7 @@ declare namespace preact {
 		componentWillUpdate?(nextProps: Readonly<P>, nextState: Readonly<S>, nextContext: any): void;
 		getSnapshotBeforeUpdate?(oldProps: Readonly<P>, oldState: Readonly<S>): any;
 		componentDidUpdate?(previousProps: Readonly<P>, previousState: Readonly<S>, snapshot: any): void;
-		componentDidCatch?(error: any): void;
+		componentDidCatch?(error: any, errorInfo: any): void;
```

Comment:
This is needed as 2 args are passed into `componentDidCatch` in React, which causes an incompatible type equality. Currently, we only pass a single argument into `componentDidCatch`

Reply:
I think we could probably pass a part of the returned error object back as the second part. Could pass through `error.message`.